e2e/gui/src/test/java/com/epam/pipeline/autotests/GlobalSearchTest.java

```diff
@@ -230,6 +249,79 @@ public void searchForPipelineWithRuns() {
                 .close();
     }
 
+    @Test(dependsOnMethods = {"searchForPipelineWithRuns"})
+    @TestCase(value = {"EPMCMBIBPC-2663"})
+    public void searchForPipelineRun() {
+        home();
+        search()
+                .click(RUNS)
+                .search(pipeline)
+                .enter()
+                .sleep(2, SECONDS)
+                .hover(SEARCH_RESULT)
+                .openSearchResultItemWithText(pipelineWithRun)
+                .ensure(TITLE, Status.SUCCESS.reached, text(testRunID), text(pipeline), text(draftVersionName))
+                .checkTags(configurationDisk, configurationNodeType)
+                .ensure(HIGHLIGHTS, text("Found in pipelineName"), text("Found in description")/*,
+                        text("Found in logs")*/)
```

Comment:
Do we need this commented out line?

Reply:
Check "Found in logs" field has been changed to check "Found in podId"